assets/scss/components/_side-navigation.scss

```diff
@@ -37,12 +37,26 @@ Styleguide Side Navigation
   .side-nav__component {
     display: none;
     @include media(tablet) {
-        display: block;
+      display: block;
     }
     &:last-child {
       display: block;
     }
   }
+  .side-nav__link {
+    padding: $gutter/1.5 $gutter $gutter/1.5 $gutter/2;
```

Comment:
As an aside - this is something that design need to talk about, `$gutter` width and how we divide it up. There are a lot of calculations here which is potentially a smell that the white space needs a look at.